testdata/p4tc_samples_outputs/digest.template

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -x
+
+set -e
+
+TC="tc"
+$TC p4template create pipeline/digest numtables 1
+
+$TC p4template create action/digest/ingress/send_nh actid 1 \
+	param port type dev \
+	param srcMac type macaddr \
+	param dstMac type macaddr
+$TC p4template update action/digest/ingress/send_nh state active
+
+$TC p4template create action/digest/ingress/drop actid 2
+$TC p4template update action/digest/ingress/drop state active
+
+$TC p4template create extern/root/Digest extid 0x05000000 numinstances 1 tc_acl 0x19b6 has_exec_method
+
+$TC p4template create extern_inst/digest/Digest/Ingress_Deparser.digest_inst instid 1 \
+control_path tc_key index ptype bit32 id 1 param srcAddr ptype bit48 id 2 param ingress_port ptype bit32 id 3
```

Comment:
Small detail, the P4 program has the extern instance param (`srcAddr`) annotated with `macaddr`, however this is not being reflected in template or JSON. `ingress_port` is also suffering from the same issue